package.json

```diff
@@ -4,7 +4,7 @@
   "description": "HTTP API for Bedrock User Accounts",
   "main": "./lib",
   "scripts": {
-    "lint": "eslint lib test"
+    "lint": "eslint **/*.js"
```

Comment:
Thanks for looking into this issue.  Does this syntax deal with nested folders and any depth?  so if we have this dir structure:
`/lib/agents/SomeAgent.js`

Reply:
checked and it does.

Reply:
Note glob info here: https://eslint.org/docs/user-guide/command-line-interface
The above will use shell globbing.  If you want node glob module then need to escape that: `"eslint '**/*.js'"`.

Reply:
@davidlehn I tested quoting the globbing and it still did not do the right thing.
My test was...
create /lib/testing/index.js (with a linting error)
from root `npm run lint`

I made a PR that worked for me.https://github.com/digitalbazaar/bedrock-account-http/pull/20

Reply:
As long as we're talking alternatives, if there are not too many files could always go the explicit route:
```
eslint `git ls-files '*.js'`
```

Reply:
@davidlehn interesting idea.  Did you perform my experiment and find a globbing pattern that works properly on your system?  It seems there is a difference in behavior between macos/osx/debian.

Reply:
oh wow never even though about other file systems had no idea. strange that it works on mac osx recursively.

Reply:
oh btw if you're not familar with staged-linters they do a good job of finding the files in a comment and running lint on them:

https://github.com/okonet/lint-staged

I know git hook linting is not popular around here, but I've used them in the past and loved it.

Reply:
@dlongley @mattcollier so basically is the sollution @mattcollier proposed good for everyone eslint test lib schema
does that work recursively? if so we can get this in today.

Reply:
@mattcollier Haven't figured out best way to do all this.  I think quoting the globs to get common node-glob behavior vs shell behavior is a start.  Previously I had things like `lib/*.js lib/**/*.js ...` because sometimes a hammer is the easiest tool.